plugins/catalog-backend-module-aws/api-report.md

```diff
@@ -43,4 +54,36 @@ export class AwsS3DiscoveryProcessor implements CatalogProcessor {
     parser: CatalogProcessorParser,
   ): Promise<boolean>;
 }
+
+// @public
+export class AwsS3EntityProvider implements EntityProvider {
+  constructor(
```

Comment:
I'd say make this constructor private and replace it with a `create` method which accepts an options object.

Reply:
currently, it is only used as part of `AwsS3EntityProviderFactory.fromConfig(...)`. I thought about putting the logic from it into it, too, but wanted to keep it cleaner and separated. As the factory uses it as return type I had to export it, too.

But I see your point. The options object is easier extensible. I will add it.

Reply:
done